dapps/react-dapp-v2/src/pages/index.tsx

```diff
@@ -108,6 +108,18 @@ const Home: NextPage = () => {
     await ping();
   };
 
+  async function emit() {
+    if (typeof client === "undefined") {
+      throw new Error("WalletConnect is not initialized");
+    }
+    
+    await client.emit({
+      topic: session?.topic || '',
+      event: { name: 'chainChanged' },
```

Comment:
Looks like build failed bc `data` is `any` but still expected as a property.

```suggestion
      event: { name: 'chainChanged', data: {} },
```